### ISSUE TITLE
is it possible to finetune from original weights?

### ISSUE THREAD

**Issue by yyfcc17** [facebookresearch/OctConv#14]
or we need to train from scratch every time?

**Comment by Hexuanfang** [comment]
hi, did you figure it out? i have the some question

**Comment by yyfcc17** [comment]
Yes, i tried to reuse the original weights of a conv net trained on MNIST dataset. when there are just two layers, and alpha set to 0.5, use the original weights to init the octave conv parameters, can recover an accuracy of 91% vs 98% of the original.
but when it comes to 3 layers of convolution, the accuracy drops to 17%.
i think it will get much worse after the network goes deeper.
so my answer is no. currently, it seems that we cannot adapt the finetune strategy to use OctaveConv.

**Comment by Hexuanfang** [comment]
thanks a million



---Original---
